### PR TITLE
fix: design review comments

### PR DESCRIPTION
**Pull request by NataliiaPodp**
## Checklist

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I have checked my code and corrected any misspellings
-   [x] My changes generate no new warnings
-   [x] My changes conform to the contributing guidelines (`CONTRIBUTING.md`)

### CODE REVIEW

File: layouts/partials/sections/event-highlights.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h2 class="heading heading--2">
                     {{ $input.subtitle }}
                 </h2>
 
-                <p class="paragraph">{{ $input.description | markdownify }}</p>
+                <p class="paragraph event-highlights-section__description--paragraph">{{ $input.description | markdownify }}</p>
```
Comment (lukket) [inline layouts/partials/sections/event-highlights.html:18]: I would consider `--paragraph` more an element than a modifier. What about `--dimmed`? Maybe even globally for paragraph (`paragraph paragraph--dimmed`).

File: assets/styles/layouts/partials/sections/event-highlights.css [hunk 2]
```diff
@@ -35,3 +35,8 @@
     justify-content: center;
     flex-wrap: wrap;
 }
+
+.event-highlights-section__description--paragraph {
+    color: var(--color-secondary-darker);
+    line-height: 1.5rem;
```
Comment (lukket) [inline assets/styles/layouts/partials/sections/event-highlights.css:41]: Is the line height modifier specific or the general style of a paragraph?

**Review by lukket — CHANGES_REQUESTED** [review]

File: assets/styles/layouts/partials/sections/event-highlights.css (hunk 2 above)
Comment (NataliiaPodp) [reply assets/styles/layouts/partials/sections/event-highlights.css:41]: renamed

File: layouts/partials/sections/event-highlights.html (hunk 1 above)
Comment (NataliiaPodp) [reply layouts/partials/sections/event-highlights.html]: fixed in the next comment

**Review by lukket — APPROVED** [review]

**Pull request merged**